thoth/storages/graph/models.py

```diff
@@ -361,13 +361,34 @@ class CVE(Base, BaseExtension):
     python_package_version_entities = relationship("HasVulnerability", back_populates="cve")
 
 
+class BuildLogAnalyzerRun(Base, BaseExtension):
+    """A class representing a single buildlogs-analyzer (build log analysis) run."""
+
+    __tablename__ = "build_log_analyzer_run"
+
+    id = Column(Integer, primary_key=True, autoincrement=True)
+    build_log_analyzer_name = Column(Text, nullable=True)
+    build_log_analyzer_version = Column(Text, nullable=True)
+    build_log_analysis_document_id = Column(Text, nullable=False)
+    datetime = Column(DateTime, nullable=False)
+    debug = Column(Boolean, nullable=False, default=False)
+    build_log_analyzer_error_reason = Column(Text, nullable=True)
+    duration = Column(Integer, nullable=True)
+    input_python_package_version_entity_id = Column(
+        Integer, ForeignKey("python_package_version_entity.id", ondelete="CASCADE")
+    )
+
+    input_python_package_version_entity = relationship(
+        "PythonPackageVersionEntity", back_populates="build_log_analyzer_run"
+    )
```

Comment:
There is missing a relation with PythonPackageVersionEntity. In that table we need to have also:
```
build_log_analyzer_run = relationship("BuildLogAnalyzerRun", back_populates="input_python_package_version_entity")```

Reply:
Thanks for pointing it out. 
Fixed it :+1: 
PTAL